README.md

```diff
@@ -34,46 +34,50 @@ solar and storage.
     conda activate hopp
```

Comment:
Can you please update the python version in the README just above here?

Reply:
Good catch!